acknowledgements.rst

```diff
@@ -0,0 +1,13 @@
+.. _documentation_home:
+
+Acknowledgements
```

Comment:
What do you think to add this on "About" page something similar to [Navigation2](https://navigation.ros.org/about/index.html)

Why not also add list of the core maintainers? This would increase credibility, as there are persons with names behind the framework.

And we can link to GH statistics for the developers, saying "thanks" to all individuals working with us

Reply:
I quite like this idea although I prefer Moveit's format: https://moveit.ros.org/about/

Reply:
opened #8